src/rv/comm/rcssserver/GameState.java

```diff
@@ -335,6 +372,29 @@ public void parse(SExp exp, WorldModel world) {
                     scoreRight = Integer.parseInt(atoms[1]);
                     playStateChanges++;
                     break;
+                case FOUL:
```

Comment:
I had created a `FoulReportedListener` but ending up scrapping it when I didn't seem to need it.